pkg/controllers/dynakube/logmonitoring/daemonset/volumes.go

```diff
@@ -121,7 +114,6 @@ func getIngestVolumes() []corev1.Volume {
 			VolumeSource: corev1.VolumeSource{
 				HostPath: &corev1.HostPathVolumeSource{
 					Path: dockerLogsVolumePath,
-					Type: ptr.To(corev1.HostPathDirectory),
```

Comment:
I think these (`ptr.To(corev1.HostPathDirectory)`) changes can stay, no?
@chrismuellner

Reply:
directories on host should not be created if they don't exist, so `type: Directory` should stay

Reply:
changed type of hostpaths to `Directory`

https://github.com/Dynatrace/dynatrace-operator/pull/4455/commits/933e760364796c6778c784149bfc789bb2f64241